lib/utils/pagination/paginationHelper.js

```diff
@@ -0,0 +1,11 @@
+const { LIMIT_HELPER, OFFSET_HELPER } = require('../../../constants');
+
+const pagination = (query) => {
+  const page = parseInt(query.offset, 10) || OFFSET_HELPER;
+  const limit = parseInt(query.limit, 10) || LIMIT_HELPER;
+  const offset = limit * (page - 1);
```

Comment:
what happens when page is less than 1?

Reply:
In that case, the offset will be equal to 0 and the query will have an offset of 0  and limit=15

Reply:
i have added a condition for checking negative and invalid integers